tools/serverpod_cli/lib/src/util/model_helper.dart

```diff
@@ -34,31 +34,17 @@ class ModelHelper {
     var modelSources = <ModelSource>[];
 
     var modelSource = await _loadYamlModelsFromDisk(
-      defaultModuleAlias,
-      _absolutePathParts(config.modelSourcePathParts),
+      moduleAlias: defaultModuleAlias,
       loadConfig: config,
+      makeConfigPathsAbsolute: true,
```

Comment:
Comment: I think this is a little dirty solution, but I also know the struggle you had with the code so perhaps we need to have a rethink about the structure here in the future.

Reply:
Fixed! Reverted back to the original code where the absolute path is configured in the call of the method.